docs/block-development/extensible-blocks/cart-and-checkout-blocks/introduction-checkout-blocks/01-understanding-checkout-blocks.md

```diff
@@ -0,0 +1,430 @@
+# Understanding WooCommerce Checkout Blocks
+
+**Reading time: 5 minutes**
+
+> **💡 Compatibility Note:** This guide covers WooCommerce 9.0+ with Checkout Blocks (bundled since WooCommerce 8.3+). If you're using an older version, some features may not be available. Check your WooCommerce version: **WooCommerce → Settings → Status**.
+
+## What This Document Covers
+
+By the end of this guide, you'll understand:
+- The fundamental architectural shift from shortcode to blocks
+- Why WooCommerce made this change
+- The three core concepts powering checkout blocks
+- Whether checkout blocks are right for your project
+
+## The Shift: From Templates to Components
+
+### Shortcode Checkout (Traditional)
```

Comment:
I like the conceptual difference, but this section could probably be streamlined a bit. Maybe remove the bullet points and just get it into one - two sentences for each approach.